### PR TITLE
Dropzone Select Action

### PR DESCRIPTION
**Pull request by manuelsalomon**

### CODE REVIEW

File: src/components/ClearingsUploader/ClearingsUploader.js [hunk 1]
```diff
@@ -43,6 +44,28 @@ const propTypes = {
...
+    or: 'o hace ',
+    click_here: 'click acá',
+    manual_text: ' y subilo manualmente',
+    uploading: 'Subiendo archivo',
+    upload_success: 'Archivo subido',
+    upload_failure: 'El archivo no subió correctamente',
+    format_error: 'El tipo de archivo no es valido',
+    retry: 'Intentar de nuevo'
+  },
+  options: [
```
Comment (santi698) [inline src/components/ClearingsUploader/ClearingsUploader.js]: This default is definitely not ok, we don't want dummy select values in production.

File: src/components/ClearingsUploader/ClearingsUploader.js [hunk 2]
```diff
@@ -109,7 +133,9 @@ export const ClearingsUploader = ({
     <ClearingsUploaderWrapper>
       <FileUploadWrapper>
         <StyledP>{messages.step_1}</StyledP>
-        <Select id="provider">{ListItems}</Select>
+        <Select id="provider" onChange={onSelect}>
```
Comment (santi698) [inline src/components/ClearingsUploader/ClearingsUploader.js]: You need to pass `value={provider}` and receive the provider as a prop to make this useful.

File: src/components/ClearingsUploader/ClearingsUploader.js (hunk 1 above)
Comment (manuelsalomon) [reply src/components/ClearingsUploader/ClearingsUploader.js]: done

File: src/components/ClearingsUploader/ClearingsUploader.js (hunk 2 above)
Comment (manuelsalomon) [reply src/components/ClearingsUploader/ClearingsUploader.js]: done

**Pull request merged**